src/components/SlateEditor/plugins/DND/index.js

```diff
@@ -1,31 +1,35 @@
-import { findNode, findRange } from 'slate-react';
+/**
+ * Copyright (c) 2017-present, NDLA.
```

Comment:
2019

Reply:
I stedet for å kommentere disse, skal vi bare search and replace alle lisens-kommentarer med 2019?